_static/api-examples.md

```diff
@@ -152,7 +152,7 @@ You can sign a manifest with either the e-Manifest ID:
 ```bash
 curl -i -X POST -H 'Content-Type: application/json' \
   --data @sign-manifest-emanifestid.json \
-  https://e-manifest.18f.gov/api/0.1/manifest/sign
+  https://e-manifest.18f.gov/api/v1/manifests/:manifest_id/sign
```

Comment:
that's why I suggested we adopt a e-Manifest id that uses UUID, so there would be little to zero risk.